tls/s2n_server_cert_request.c

```diff
@@ -127,10 +150,22 @@ int s2n_client_cert_req_recv(struct s2n_connection *conn)
 }
 
 
-int s2n_client_cert_req_send(struct s2n_connection *conn)
+int s2n_server_cert_req_send(struct s2n_connection *conn)
 {
     struct s2n_stuffer *out = &conn->handshake.io;
 
+    if (conn->actual_protocol_version >= S2N_TLS13) {
```

Comment:
This shares no code in common with the tls1.2 version -- it should probably be a new, separate handler.